Example/Kommunicate_ExampleUITests/KommunicateRichMessageUITests.swift

```diff
@@ -82,15 +96,30 @@ class KommunicateRichMessageUITests: XCTestCase {
         app.typeText(GroupData.typeText4) // typing message
         app.buttons[InAppButton.ConversationScreen.send].tap() // sending message in group
         let innerchatscreentableviewTable = app.tables[AppScreen.innerChatScreenTableView]
+        guard innerchatscreentableviewTable.staticTexts[RichMessageButtons.submitButton].isEnabled else {
+            XCTFail("Button is disabled or not visible")
+            return
+        }
         innerchatscreentableviewTable.staticTexts[RichMessageButtons.submitButton].tap()
+        print("Different button 1 Rich message Pass")
+        guard innerchatscreentableviewTable.staticTexts[RichMessageButtons.suggestedReplyButton].isEnabled else {
+            XCTFail("Button is disabled or not visible")
+            return
+        }
         innerchatscreentableviewTable.staticTexts[RichMessageButtons.suggestedReplyButton].tap()
+        print("Different button 2 Rich message Pass")
```

Comment:
Same as above: change this to "Tapped..."

Reply:
done